example/burrow_integration/get_transactions.py

```diff
@@ -0,0 +1,199 @@
+        params = params + integration_helpers.left_padded_address_of_param(x, no_of_param)
+    params = params + integration_helpers.argument_encoding(ADMIN_ACCOUNT_ID)  # account id
+    params = params + integration_helpers.argument_encoding("coin#test")  # asset id
+    params = params + integration_helpers.argument_encoding('10')  # page size
+    params = params + integration_helpers.argument_encoding('')  # first_tx_hash
+    params = params + integration_helpers.argument_encoding('')  # ordering
+    params = params + integration_helpers.argument_encoding('')  # first_tx_time
+    params = params + integration_helpers.argument_encoding('')  # last_tx_time
+    params = params + integration_helpers.argument_encoding('')  # first_tx_height
+    params = params + integration_helpers.argument_encoding('')  # last_tx_height
+    tx = iroha.transaction(
+        [
+            iroha.command(
+                "CallEngine", caller=ADMIN_ACCOUNT_ID, callee=address, input=params
+            )
+        ]
+    )
+    IrohaCrypto.sign_transaction(tx, ADMIN_PRIVATE_KEY)
+    response = net.send_tx(tx)
+    print(response)
+    for status in net.tx_status_stream(tx):
+        print(status)
+    hex_hash = binascii.hexlify(IrohaCrypto.hash(tx))
+    return hex_hash
+
+@integration_helpers.trace
+def get_pending_transactions(address, first_tx_time, last_tx_time):
+    params = integration_helpers.get_first_four_bytes_of_keccak(b'getPendingTransactions(string,string,string,string,string)')
+    no_of_param = 5
+    for x in range(no_of_param):
+        params = params + integration_helpers.left_padded_address_of_param(x, no_of_param)
+    params = params + integration_helpers.argument_encoding('10')  # page size
+    params = params + integration_helpers.argument_encoding('')  # first_tx_hash
+    params = params + integration_helpers.argument_encoding('')  # ordering
+    params = params + integration_helpers.argument_encoding(str(first_tx_time))  # first_tx_time
+    params = params + integration_helpers.argument_encoding(str(last_tx_time))  # last_tx_time
+    tx = iroha.transaction(
+        [
+            iroha.command(
+                "CallEngine", caller=ADMIN_ACCOUNT_ID, callee=address, input=params
+            )
+        ]
+    )
+    IrohaCrypto.sign_transaction(tx, ADMIN_PRIVATE_KEY)
+    response = net.send_tx(tx)
+    print(response)
+    for status in net.tx_status_stream(tx):
+        print(status)
+    hex_hash = binascii.hexlify(IrohaCrypto.hash(tx))
+    return hex_hash
+
+@integration_helpers.trace
+def get_transaction(address):
+    params = integration_helpers.get_first_four_bytes_of_keccak(b'getTransaction(string)')
+    no_of_param = 1
+    for x in range(no_of_param):
+        params = params + integration_helpers.left_padded_address_of_param(x, no_of_param)
+    params = params + integration_helpers.argument_encoding('67619a99012600432ac15225373d192dba4b6d5f47dbf89daa8ec9fd9bee8c48')  # transaction hash 
```

Comment:
Transaction hash can change and even would change for sure, so I suggest to add extra param to the function